servers/zms/schema/zms_server.sql

```diff
@@ -128,6 +128,7 @@ CREATE TABLE IF NOT EXISTS `zms_server`.`service` (
   `svc_group` VARCHAR(256) NOT NULL DEFAULT '',
   `modified` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3) ON UPDATE CURRENT_TIMESTAMP(3),
   `created` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+  `description` VARCHAR(4096) NOT NULL,
```

Comment:
can we also please set this with default empty string to be consistent with other fields.